packages/patternfly-4/react-core/src/components/Wizard/Wizard.md

```diff
@@ -820,3 +820,33 @@ class SimpleWizard extends React.Component {
   }
 }
 ```
+
+### Wizard - in page
+
+```js
+import React from 'react';
+import { Wizard } from '@patternfly/react-core';
+
+class InPageWizard extends React.Component {
+  render() {
+    const steps = [
+      { name: 'Step 1', component: <p>Step 1</p> },
+      { name: 'Step 2', component: <p>Step 2</p> },
+      { name: 'Step 3', component: <p>Step 3</p> },
+      { name: 'Step 4', component: <p>Step 4</p> },
+      { name: 'Final Step', component: <p>Final Step</p>, hideCancelButton: true, nextButtonText: 'Close' }
+    ];
+
+    return (
+      <div style={{height: '400px'}}>
+        <Wizard
+          isInPage
+          isOpen
```

Comment:
I kept the behaviour of `isOpen`. Meaning one has to always specify it when using in-page wizard. Otherwise nothing is rendered.

Should this behaviour be changed and render the in-page wizard always?

Reply:
I think it makes sense to make isOpen the default in the code if isInPage is true. Just like any other in page components they should show by default, if user needs to toggle visibility they can do so on their end

Reply:
done